packages/components/src/side_scroller/index.jsx

```diff
@@ -169,6 +173,8 @@ const SideScroller = ({ className: passedClassName, children, ...cleanProps }) =
 SideScroller.propTypes = {
   className: PropTypes.string,
   children: PropTypes.object,
+  onShift: PropTypes.func,
```

Comment:
I'm a bit unhappy with the prop name because it gives the impression to always be called when the position inside the scroller changes but it's actually only called if user clicks on one of the arrows.

Similar situation here with `onDragStart`. Wouldn't it be better to have something like `onPositionChange` that covers both cases?

Reply:
I thought that in the beginning as well, but we have 2 cases specified in one trigger (because user can interact with the sidescroller two ways - by pressing buttons and sliding, and it is, well, to different things, imo), so it seemed to be a bit more clear. 
On the other hand - yeah, the case is only one. 

And I've just realised that interaction in this case can actually be tracked via clicking on the block itself, without specifiying is it drag or is it shift.